src/Agent.Listener/Agent.cs

```diff
@@ -194,13 +194,14 @@ private void CtrlCHandler(object sender, EventArgs e)
                 bool autoUpdateInProgress = false;
                 Task<bool> selfUpdateTask = null;
                 jobDispatcher = HostContext.CreateService<IJobDispatcher>();
+
                 while (!token.IsCancellationRequested)
```

Comment:
Gotcha, just wanted to make sure it wasn't an oversight. You might want to add a comment.